packages/platform-fastify/package.json

```
@@ -17,16 +17,26 @@
     "access": "public"
   },
   "dependencies": {
-    "fastify": "3.29.0",
-    "fastify-cors": "6.1.0",
-    "fastify-formbody": "5.3.0",
+    "@fastify/cors": "7.0.0",
```

v7 supports fastify v3

```suggestion
    "@fastify/cors": "8.0.0",
```